src/digits/ff31.rs

```diff
@@ -534,6 +535,12 @@ macro_rules! fp31 {
             }
 
             impl $classname {
+                /// Return true if the $classname represents an even value.
+                pub fn is_even(&self) -> bool {
```

Comment:
You don't need this.

Reply:
Correct. We didn't end up using it, but figured it might be useful in general. If not, it can certainly be removed.

Reply:
We can't have anything that returns a bool if people did need it, because if/else would cause jumps. Also any algorithm that needs to check this will not be constant time.